.semaphore/semaphore.yml

```diff
@@ -0,0 +1,175 @@
+version: v1.0
+name: CI Pipeline
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu2004
+blocks:
+  - name: Test Bootstrap
+    dependencies: []
+    run:
+      when: change_in('/bootstrap/')
+    task:
+      jobs:
+        - commands:
+            - checkout
+            - cd bootstrap
+            - go test -v -coverprofile=coverage.out ./...
+            - go tool cover -html=coverage.out -o coverage.html 
+          name: Test Bootstrap
+  - name: Test certs

```

Comment:
add tests for `cli`, `coap`, `consumers`, whole `http` sub package not only `/api`, and others

Reply:
On it

Reply:
Stay consistent and use capital letter for the name of each service (Certs, Things, ...)